app/lib/rummager_api.rb

```diff
@@ -0,0 +1,7 @@
+require "gds_api/rummager"
+
+class RummagerApi
+  def self.instance
+    @rummager_api ||= GdsApi::Rummager.new(Plek.new.find("search"))
+  end
+end
```

Comment:
A pattern that other gov.uk apps have been using is a Services module that has class methods on it for each gds-api used by the app.  Might be worth pushing this as far as that too.  For example: https://github.com/alphagov/policy-publisher/blob/0f2fa349a043ae7acea37e22bbb08b57154fb30e/lib/services.rb